README.md

```diff
@@ -25,6 +26,18 @@ Once installation is done, you can start the app with
 npm start
 ```
 
+Once the 🐶 Hoodie app is running, you can open your browser to
+
+```
+localhost:8080
```

Comment:
maybe we can add the link directly? 

`Once the 🐶 Hoodie app is running, you can open your browser to [localhost:8080](http://localhost:8080).`